src/rpcnet.cpp

```diff
@@ -165,6 +173,18 @@ UniValue getpeerinfo(const UniValue& params, bool fHelp)
         }
         obj.push_back(Pair("whitelisted", stats.fWhitelisted));
 
+        UniValue sendPerCmd(UniValue::VOBJ);
+        BOOST_FOREACH( const mapCmdSize::value_type &i, stats.mapSendBytesPerCmd) {
+            sendPerCmd.push_back(Pair(i.first, i.second));
+        }
+        obj.push_back(Pair("bytessent_per_cmd", sendPerCmd));
+
+        UniValue recvPerCmd(UniValue::VOBJ);
+        BOOST_FOREACH( const mapCmdSize::value_type &i, stats.mapRecvBytesPerCmd) {
+            recvPerCmd.push_back(Pair(i.first, i.second));
```

Comment:
I also though about that, but I think the null entries are better instead of hiding. It result in more controllable parsing.